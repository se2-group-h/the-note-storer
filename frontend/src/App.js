//import logo from './logo.svg';
import './App.css';
import Home from './pages/home.js';
import Signup from './pages/signup.js';
import {BrowserRouter as Router, Route, Switch} from 'react-router-dom';

function App() {
  return (
    <Router>
    <div>
    <Switch>
        <Route path exact="/" component={Home} />
        <Route path="/register" component={Signup} />
    </Switch>
    </div>
    </Router>
  )
}

export default App;

