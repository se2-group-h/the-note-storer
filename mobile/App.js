import { StatusBar } from 'expo-status-bar';
import React, {useEffect, useState } from 'react';
import { StyleSheet, Text, TouchableOpacity, View, TextInput, Button  } from 'react-native';
import ComponentName from './ComponentName';

export default function App() {

  const [isPressed, setIdPressed] = useState(false);
  const [text, setText] = useState('');

  useEffect(() => {
      fetch('https://jsonplaceholder.typicode.com/todos/1')
      .then(resp => resp.json())
      .then(json => setText(JSON.stringify(json)));
  });

  return (
    <View style={styles.container}>
      <TextInput placeholder="Login" maxLength={15}/>
      <TextInput placeholder="Password" maxLength={15}/>
      <View style={{backgroundColor:"yellow", width:15}}>
        <Button title="Log in"/>
      </View>
      <Button title="Register"/>
      <Text backgroundColor="green">Zdarova!</Text>
      <StatusBar style="auto" />
      <TouchableOpacity onPress={() => setIdPressed(!isPressed)}> 
      <Text>Press me</Text>
      </TouchableOpacity>
      {/* {
        isPressed && <Text>{text ?? "null"}</Text>
      } */}
      {/* <Text>{text}</Text> */}
      <Text>{isPressed.toString()}</Text>
      <ComponentName isPressed={isPressed}/>
    </View>
  )};

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#fff',
    alignItems: 'center',
    justifyContent: 'center',
  },
});
