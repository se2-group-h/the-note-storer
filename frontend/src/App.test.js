import {render, screen} from '@testing-library/react';
import App from './App';

/*test('renders learn react link', () => {
  render(<App />);
  const linkElement = screen.getByText(/learn react/i);
  expect(linkElement).toBeInTheDocument();
});*/

test('should take a snapshot', () => {
  render(<App />);

  screen.debug();

  expect(screen.getByText('RecipeIT')).toBeInTheDocument();
});