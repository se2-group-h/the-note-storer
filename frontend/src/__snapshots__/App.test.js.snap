// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div>
    <div>
      <div>
        <header>
          <i
            aria-hidden="true"
            class="fa fa-sticky-note"
          />
          <h2>
            noteit
          </h2>
          <nav>
            <ul
              class="nav__links"
            >
              <li>
                <strong>
                  About
                </strong>
              </li>
              <li>
                <strong>
                  Contact Us
                </strong>
              </li>
            </ul>
          </nav>
          <a
            class="cta"
            href="/#"
          />
          <a
            href="/register"
          >
            <button
              id="register"
              title="signupbtn"
            >
              Register
            </button>
          </a>
          <a
            class="cta"
            href="/#"
          />
          <a
            href="/signin"
          >
            <button
              id="sign-in"
              title="signinbtn"
            >
              Sign-In
            </button>
          </a>
        </header>
      </div>
      <div
        id="body"
      />
    </div>
  </div>
</DocumentFragment>
`;
